qemu/tests/qemu_guest_agent_snapshot.py

```diff
@@ -55,18 +55,24 @@ def _action_after_fsthaw(self, *args):
         if self.bg:
             self.bg.join()
         # Make sure the returned file is identical to the original one
-        self.host_path_returned = "%s-returned" % self.host_path
-        self.vm.copy_files_from(self.guest_path, self.host_path_returned)
-        error_context.context("comparing hashes", logging.info)
-        self.curr_hash = crypto.hash_file(self.host_path_returned)
-        if self.orig_hash != self.curr_hash:
-            self.test.fail("Current file hash (%s) differs from "
-                           "original one (%s)" % (self.curr_hash,
-                                                  self.orig_hash))
-
-        error_context.context("Reboot and shutdown guest.")
-        self.vm.reboot()
-        self.vm.destroy()
+        try:
+            self.host_path_returned = "%s-returned" % self.host_path
+            self.vm.copy_files_from(self.guest_path, self.host_path_returned)
+            error_context.context("comparing hashes", logging.info)
+            self.curr_hash = crypto.hash_file(self.host_path_returned)
+            if self.orig_hash != self.curr_hash:
+                self.test.fail("Current file hash (%s) differs from "
+                               "original one (%s)" % (self.curr_hash,
+                                                      self.orig_hash))
```

Comment:
@xiagao I don't find reboot and shutdown guest in polarion.  But your test steps included this action.  If don't need this test.  please remove this item. or add this step to polarion. 8) Reboot and shutdown guest.

Reply:
@xfu check from polarion,there is no this step, delete it in docstring.
Pls have a check.
Thanks.